app/javascript/admin/controllers/config_controller.js

```diff
@@ -46,10 +48,18 @@ export default class ConfigController extends Controller {
     }
   }
 
-  closeAdminConfigModal() {
+  closeAdminModal() {
+    const authSectionUpdateConfigBtn = this.authSectionFormTarget.querySelector(
+      'input[type="submit"]',
+    );
+
     this.configModalAnchorTarget.innerHTML = '';
     document.body.style.height = 'inherit';
     document.body.style.overflowY = 'inherit';
+
+    if (authSectionUpdateConfigBtn.hasAttribute('disabled')) {
+      authSectionUpdateConfigBtn.removeAttribute('disabled');
+    }
```

Comment:
Since this is only assigned to a `const` in the context of this function, you probably can get away with making this variable less verbose/less specific:

```suggestion
  closeAdminModal() {
    const submitButton = this.authSectionFormTarget.querySelector(
      'input[type="submit"]',
    );

    this.configModalAnchorTarget.innerHTML = '';
    document.body.style.height = 'inherit';
    document.body.style.overflowY = 'inherit';

    if (submitButton.hasAttribute('disabled')) {
      submitButton.removeAttribute('disabled');
    }
```